### PR TITLE
Pizza delivery event now has a chance to add nanomachines to a pizza

### PR DESCRIPTION
**Pull request by Cyberboss**
:cl:
add: More interesting recipes to the pizza delivery event
/:cl:

### CODE REVIEW

File: code/modules/events/shuttle_loan.dm [hunk 1]
```diff
@@ -175,8 +175,11 @@
 				shuttle_spawns.Add(/obj/item/pizzabox/meat)
 				shuttle_spawns.Add(/obj/item/pizzabox/meat)
 				shuttle_spawns.Add(/obj/item/pizzabox/vegetable)
-				if(prob(10))
-					shuttle_spawns.Add(/obj/item/pizzabox/bomb)
+				if(prob(20))
+					if(prob(50))
+						shuttle_spawns.Add(/obj/item/pizzabox/bomb)
+					else
+						shuttle_spawns.Add(/obj/item/pizzabox/margherita/robo)
```
Comment (ninjanomnom) [inline code/modules/events/shuttle_loan.dm]: It'd probably be a decent idea to convert this all to 6 picks from a weighted list

**Comment by JohnOxford** [comment]
ahem.


**_### NANOMACHINES SON_**

**Comment by Time-Green** [comment]
nanites should convert their biomass into pizza

File: code/modules/events/shuttle_loan.dm (hunk 1 above)
Comment (ExcessiveUseOfCobblestone) [reply code/modules/events/shuttle_loan.dm]: Just make it pure RNG sissies

```dm
//tab to be under if(PIZZA_DELIVERY)
var/naughtypizza = list(/obj/item/pizzabox/bomb,/obj/item/pizzabox/margherita/robo) //oh look another blaklist, for pizza nonetheless!
var/nicepizza = (subtypesof(/obj/item/pizzabox) - naughtypizzas)
var/gimmepizza
for(var/i in 1 to 6)
    gimmepizza = pick(nicepizza)
    shuttle_spawns.Add(gimmepizza)
if(prob(20))
    gimmepizza = pick(naughtypizza)
    shuttle_spawns.Add(gimmepizza)
```

**Comment by Tlaltecuhtli** [comment]
is this a reference or something

**Comment by Therapist1** [comment]
Dear god the pizza crate now has 3 fucking things that can be in it besides pizza

File: code/modules/events/shuttle_loan.dm (hunk 1 above)
Comment (Cyprex) [reply code/modules/events/shuttle_loan.dm]: That's 6 pizzas+However many naughtypizzas were picked, so you immediately know how many borked pizzas you got.

**Comment by Cyberboss** [comment]
If you aren't paying, you are the product

**Comment by Jalleo** [comment]
add a one in a million chance that you instead get a dead monkey wearing a pizza shirt. With flavour text saying that pizza crumbs cover its mouth.

**Comment by ishitbyabullet** [comment]
Aren't Nanomachines able to turn you into a cyborg?

Feels like it would just be powergamed.

**Comment by ExcessiveUseOfCobblestone** [comment]
you can't powergame something you have no control over.

**Pull request merged**